{"raw":{},"required":["Filename","Size"],"package":"python3-pytest-instafail","source":"pytest-instafail","version":"0.4.2-1","section":"universe/python","priority":"optional","architecture":"all","depends":["python3-pytest","python3:any"],"suggests":["python-pytest-instafail-doc"],"installedSize":34,"maintainer":"Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>","description":"plugin for py.test that shows failures and errors instantly","homepage":"https://pypi.python.org/pypi/pytest-instafail","filename":"pool/universe/p/pytest-instafail/python3-pytest-instafail_0.4.2-1_all.deb","size":6016,"md5":"f55822190b7516656cf81bce5145757e","sha1":"8bc5cc663c81c81d547b678afc0493ced0e34240","sha256":"cb14baee249aee80f9bea836420e08a78dadc097b63084a2c0ce754e8266982c","sha512":"c447fc63d5746afbfe6df73179e2f3f66a359ff50670e03963ef0a5513a55c1375050903e407af8fb90ef6b7c93d3f13f1ce28fae397d42737231b16c81734cb","descriptionMd5":"4acabc61b3293224ea4a27802dc75041"}